src/linpred.jl

```diff
@@ -181,6 +200,13 @@ model_response(obj::LinPredModel) = obj.rr.y
 
 fitted(m::LinPredModel) = m.rr.mu
 predict(mm::LinPredModel) = fitted(mm)
+
+"""
+Extract the formula from a model.
+"""
 formula(obj::LinPredModel) = ModelFrame(obj).formula
+"""
```

Comment:
I don't think you need to repeat this here either, as the definition from StatsBase is enough.